### PR TITLE
B-20527-INT Add moves.assigned cols, create queue management FF

### PR DESCRIPTION
**Pull request by paulstonebraker**
## Summary

This PR adds columns to the moves table to track the assigned office_user, and creates a feature flag for queue management.

### CODE REVIEW

File: config/flipt/storage/development.features.yaml [hunk 1]
```diff
@@ -113,6 +113,14 @@ flags:
   - segment:
       key: mil-app
       value: false
+- key: queue_management
+  name: Office user queue management feature flag
+  type: BOOLEAN_FLAG_TYPE
+  enabled: false
```
Comment (pambecker) [inline config/flipt/storage/development.features.yaml:119]: I believe we've always set these to true in this file, I guess so it's ready to go for our dev environments.

File: migrations/app/schema/20240718021218_add_assigned_cols_to_moves.up.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+ALTER TABLE moves
+    ADD COLUMN sc_assigned_id uuid
```
Comment (pambecker) [inline migrations/app/schema/20240718021218_add_assigned_cols_to_moves.up.sql:2]: These add columns need the 'IF NOT EXISTS'

File: config/flipt/storage/development.features.yaml (hunk 1 above)
Comment (paulstonebraker) [reply config/flipt/storage/development.features.yaml]: Updated

**Review by pambecker — APPROVED** [review]

**Review by joeydoyecaci — APPROVED** [review]

File: pkg/models/move.go [hunk 3]
```diff
@@ -92,6 +92,12 @@ type Move struct {
 	LockExpiresAt                *time.Time            `json:"lock_expires_at" db:"lock_expires_at"`
 	AdditionalDocumentsID        *uuid.UUID            `json:"additional_documents_id" db:"additional_documents_id"`
 	AdditionalDocuments          *Document             `belongs_to:"documents" fk_id:"additional_documents_id"`
+	SCAssignedID                 *uuid.UUID            `json:"sc_assigned_id" db:"sc_assigned_id"`
+	SCAssigned                   *OfficeUser           `belongs_to:"office_users" fk_id:"sc_assigned_id"`
```
Comment (pambecker) [inline pkg/models/move.go:96]: Is this going to be the person's name field?  If so, could we add 'User' maybe to be more clear that it's holding a name?
i.e. SCAssignedUser, TOOAssignedUser, TIOAssignedUser

Comment (paulstonebraker) [reply pkg/models/move.go:96]: Yep I like it

**Comment by pambecker** [comment]
looks good, ready to go!

**Pull request merged**